src/ccache/config.cpp

```diff
@@ -71,6 +71,7 @@ namespace {
 
 enum class ConfigItem {
   absolute_paths_in_stderr,
+  atfile_format,
```

Comment:
Since MSVC calls these files "response files" and GCC AFAIK doesn't have a specific name for them, maybe it would be better to call the setting `response_file_format`?

Reply:
Let me know if you also want me to change `enum class AtFileFormat` to `enum class ResponseFileFormat`.

Let me know if you want me to change the public options from `gcc` and `msvc` to `posix` and `windows`. I prefer `posix` and `windows` to match the clang `--rsp-quoting=[posix|windows]` option since we actually just care about the quoting behaviour of the response file. We don't care about the compiler default quoting behaviour.

Reply:
> Let me know if you also want me to change `enum class AtFileFormat` to `enum class ResponseFileFormat`.

You can leave it for now if you like.

> Let me know if you want me to change the public option values from `gcc` and `msvc` to `posix` and `windows`.

Yes, that sounds good.